### DISCUSSION TITLE
Dask installation on DataBricks

### DISCUSSION THREAD

**Discussion by Nikhil1291** [dask/dask#9877] [General]
#! /bin/bash
/databricks/python/bin/pip install dask distributed --upgrade
echo $DB_IS_DRIVER
if [[ $DB_IS_DRIVER == "TRUE"]]; then
   dask-scheduler &>/dev/null & echo $!
else
   dask-worker tcp://$DB_IS_DRIVER_IP:8786  &>/dev/null & echo $!
fi

tried enabling the global init script but cannot initialize the cluster, please advise

**Comment by ncclementi** [comment]
hi @Nikhil1291  , Dask has a [Discourse forum](https://dask.discourse.group/t/introduce-yourself/22/23) for usage-questions.

We'll see you there! :)
